src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -542,14 +538,14 @@ class NetworkOPsImp final
     clock_type& m_clock;
     beast::Journal m_journal;
 
-    std::unique_ptr <LocalTxs> m_localTX;
+    std::unique_ptr <LocalTxs> m_localTX {make_LocalTxs()};
```

Comment:
I'd personally prefer to leave this in the original place. Moving it here gets us very little, and may even make the code a bit harder to read.